cypress/integration/finance/create_GL_journal_type_spec.js

```diff
@@ -1,3 +1,25 @@
+let description;
+let category;
+let postingType1;
+let currency;
+let docType;
+let postingType2;
+let postingType3;
+let postingType4;
+
+it('Read fixture and prepare the names', function() {
+  cy.fixture('finance/create_GL_journal_type_spec.json').then(f => {
+    description = f['description'];
+    category = f['category'];
+    postingType1 = f['postingType1'];
+    currency = f['currency'];
+    docType = f['docType'];
+    postingType2 = f['postingType2'];
+    postingType3 = f['postingType3'];
+    postingType4 = f['postingType4'];
+  });
+});
+
 describe('Creating GL journal entry and change posting type', function() {
   before(function() {
     cy.visit('/window/540356');
```

Comment:
Instead of `cy.visit('/window/540356');  cy.clickHeaderNav(Cypress.messages.window.new.caption);`, we could do  `cy.visitWindow('540356', "NEW")`

Reply:
done.